modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -158,278 +160,287 @@ Deploying myMuleApp ...
+If you create and test an application in a sandbox environment, you must move it to the 
+production environment before deploying it.
+You must have the CloudHub Admin role to move applications between environments.
 
-image::deploying-to-cloudhub-update.png[]
+To avoid domain name conflict, an application with the same name cannot be deployed in two environments
+at the same time.
+You must modify the application name to deploy it to another environment.
 
-[[worker_sizing]]
-=== Worker Sizing
+Runtime Manager prevents you from duplicating applications in a single environment, so you cannot move an application into the environment it's already in.
+To deploy the same or similar application in a single environment, modify the filename of one application.
 
-After deploying your application, you can allocate the number and size of the worker associated with your application. On each application, workers are responsible for executing the application logic.
+To move an application from sandbox to production:
 
-[TIP]
---
-Only running applications count toward workers usage. +
-`Stopped` applications do not consume vCore availability.
---
+. Sign in to Anypoint Platform.
+. Select *Runtime Manager* and change to your production environment.
+. In the *Applications* page, click *Deploy application*.
++
+You see the *Deploy Application* page:
++
+.The screenshot shows (*1*) the *Application Name* field, (*2*) the *Get from sandbox* button, and (*3*) the *Deploy Application* button on the *Deploy Application* page.
+image::cloudhub-deploy-app-sandbox.png[Application Name field and Get from sandbox and Deploy Application buttons]
+. Provide a name for your application and click *Get from sandbox*:
++
+This button is available only if a non-production environment exists.
+. Select your sandbox environment and select the application to move to production.
++
+Only applications that are deployed to CloudHub appear in the list.
+You can promote only applications deployed to CloudHub to run on other CloudHub workers.
+. If you want to copy the environment variables and Mule version from the source application, select *Merge environment variables and mule version*.
+. Click *Apply*.
+. Click tabs to configure application options.
++
+See <<configure-application-settings,Configure Application Settings>>.
+. Click *Deploy Application*.
++
+See <<Deployment Progress>>.
 
-Worker sizes have different compute and memory capacities, including:
+== Application Names 
 
-[%header,cols="2*a"]
-|===
-|Worker Size |Heap Memory
-|0.1 vCores |500 MB
-|0.2 vCores |1 GB
-|1 vCore |1.5 GB
-|2 vCores |3.5 GB
-|4 vCores |7.5 GB
-|8 vCores |15 GB
-|16 vCores |32 GB
-|===
+The application name identifies your application in Runtime Manager and also used for the 
+public application name in the `cloudhub.io` domain.
+For example, an application named `myapplication` is accessible at `+http://myapplication.cloudhub.io+`.
```

Comment:
Not sure this is nec.

Reply:
This is already in the Move an Application text; do we need it (fully explained) in both locations?
Can we say something about a name having to be unique not only across all of cloudhub.io (external environments) but also among your internal environments?

Reply:
It's not there, though. I think the churn might have confused.